### ISSUE TITLE
Dropdown selector not correctly accessible

### ISSUE THREAD

**Issue by maecatnou** [polylang/polylang#1027]
Issue raised by client though Helscout issue => https://secure.helpscout.net/conversation/1857320988/19125?folderId=4673451

## Bug noticed
The dropdown selector isn't accessible as it should be. 
It appears to be missing the lang attributes and, according to the customer, should look more like what is referenced here => https://codyhouse.co/blog/post/accessible-language-picker#html-structure

## Side note
Greg noticed that there was some existing code that should do this but for the time being isn't likely to work due to how it is written.
Check => https://github.com/polylang/polylang/blob/master/include/walker-dropdown.php#L38=
